src/nl/surf/eduhub_rio_mapper/metrics.clj

```diff
@@ -5,13 +5,15 @@
   {:pre [(map? queue-count)
          (every? string? (keys queue-count))
          (every? integer? (vals queue-count))]}
-  (str/join "\n" (map (fn [[k v]] (format "active_and_queued_job_count{schac_home=\"%s\"} %s" k v))
+  (str/join "\n" (map (fn [[k v]] (format "rio_mapper_active_and_queued_job_count{schac_home=\"%s\"} %s" k v))
                       queue-count)))
 
-(defn count-queues [grouped-queue-counter]
+(defn count-queues [grouped-queue-counter client-schac-homes]
   {:post [(map? %)
           (every? string? (keys %))
           (every? integer? (vals %))]}
-  (merge-with +
-              (grouped-queue-counter :queue)
-              (grouped-queue-counter :busy-queue)))
+  (reduce (fn [m k] (assoc m k (get m k 0)))
```

Comment:
zet ajb wel een comment bij die reduce want dit is niet erg duidelijk.

Reply:
ok